intel_extension_for_transformers/llm/runtime/graph/models/gptj/gptj.cpp

```diff
@@ -122,6 +122,22 @@ static bool gptj_model_eval_internal(model_context& lctx, const model_token* tok
     memcpy(static_cast<model_token*>(embd->data) + i * N, tokens + i * N, N * ne_element_size(embd));
   }
 
+  bool enable_tp = false;
+#ifdef NE_TP_MODEL
+  parallel_context* p_ctx = init_parallel_context();
+  int32_t world_size = get_tp_size(p_ctx);
+  int32_t rank = get_tp_rank(p_ctx);
+  enable_tp = world_size > 1 ? true : false;
+
+  // int32_t world_size = 2;
+  // int32_t rank = 0;
+  // bool enable_tp =true;
+  if (enable_tp) {
+    // need to broadcast the ids
+    broadcast(p_ctx, (float*)embd->data,  N * batch_size * ne_element_size(embd));
+  }
```

Comment:
Is it necessary to broadcast the embd data? I think all ranks should have the same embed upon here.